File: desafio-02/leonelsanchesdasilva/csharp/desafio02.cs
using System;

namespace OsProgramadores {
    class Desafio02 {          
        static void Main(string[] args) {
            for (int i = 1; i <= 10000; i++) {
                bool primo = true;
                for (int j = 2; j <= Math.Floor(Math.Sqrt(i)); j++) {
                    if (i % j == 0) {
                        primo = false;
                        break;
                    }
                }

                if (primo) {
                    Console.WriteLine(i);
                }
            }
        }
    }
}

File: desafio-02/leonelsanchesdasilva/php/desafio02.php
<?php
    for ($i = 1; $i <= 10000; $i++) {
        $primo = True;
        for ($j = 2; $j <= floor(sqrt($i)); $j++) {
            if ($i % $j == 0) {
                $primo = False;
            }
        }
        if ($primo) {
            echo $i . ' ';
        }
    }
?>